src/envo/envo-edit.owl

```diff
@@ -39742,6 +39745,51 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> <http://purl.obolibrar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000066> <http://purl.obolibrary.org/obo/ENVO_00000015>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_01001497>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105008> (plastic fragment)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105008> "A piece of plastic which 1) is hard and 2) has an irregular or broken shape that implies it's being broken down from a large piece of litter."@en)
```

Comment:
>A piece of plastic which 1) is hard and 2) has an irregular or broken shape that implies it's being broken down from a large piece of litter.


I'd simplify and rephrase to:

> A piece of plastic which 1) is hard, 2) has an irregular shape, and 3) has broken off of a larger piece of plastic.

Also, why does it need to be hard? Is that a GESAMP constraint?

Reply:
@pbuttigieg yes, the GESAMP specify it is a hard particle. The original description is 

> irregular shaped hard particles having appearance of being broken down from a larger piece of litter

Reply:
@meichen-liu please make a note in the comment that we are being stricter with the "broken off" aspect of this. Having the appearance of being broken down (sic) is dangerous.

Reply:
@pbuttigieg we have added a comment to clarify that aspect.

> To avoid ambiguity, we are being stricter with the "has broken off of a larger piece of plastic" aspect compared to the GESAMP report, which originally stated "having appearance of being broken down from a larger piece of litter".